bin/ui5.js

```diff
@@ -26,16 +26,31 @@ if (
 	console.log("=====================================================================");
 	process.exit(1);
 } else {
-	if (semver && semver.prerelease(nodeVersion)) {
-		console.log("====================== UNSTABLE NODE.JS VERSION =====================");
-		console.log("You are using an unstable version of Node.js");
-		console.log("Detected Node.js version " + nodeVersion);
-		console.log("");
-		console.log("=> Please note that an unstable version might cause unexpected");
-		console.log("   behavior. For productive use please consider using a stable");
-		console.log("   version of Node.js! For the release policy of Node.js, see");
-		console.log("   https://nodejs.org/en/about/releases");
-		console.log("=====================================================================");
+	if (semver) {
+		if (semver.prerelease(nodeVersion)) {
+			console.log("====================== UNSTABLE NODE.JS VERSION =====================");
+			console.log("You are using an unstable version of Node.js");
+			console.log("Detected Node.js version: " + nodeVersion);
+			console.log("");
+			console.log("=> Please note that an unstable version might cause unexpected");
+			console.log("   behavior. For productive use please consider using a stable");
+			console.log("   version of Node.js! For the release policy of Node.js, see");
+			console.log("   https://nodejs.org/en/about/releases");
+			console.log("=====================================================================");
+		} else if (semver.satisfies(nodeVersion, "< 14.16 || ^15 || ^17")) {
```

Comment:
`< 14.16` means also node version `10` and `12`, correct?

Reply:
Yes:
```
> s.satisfies("10.0.0", "< 14.16")
true
```